dev_apps/synchronizer/manifest.webapp

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "Synchronizer",
```

Comment:
Hm, but this naming decision is different than for the build option, I think. If we call the app Firefox Sync now, then we have to change it again as soon as we add the first non-Syncto collection. How about 'Data Sync'? That is descriptive when it shows up in an 'active processes' list, and is more agnostic in that it can cover Syncto, Kinto, CardDav, ...